treeherder/etl/perf.py

```diff
@@ -127,6 +127,11 @@ def _load_perf_artifact(project_name, reference_data, job_data, job_guid,
                     'framework': framework,
                     'extra_properties': extra_properties,
                     'lower_is_better': subtest.get('lowerIsBetter', True),
+                    'should_alert': subtest.get('shouldAlert'),
```

Comment:
Actually wait, this was deliberate. :) These properties can be null, which indicates no preference was set. Since this is clearly non-obvious, I'll add some comments explaining.